frontend/src/components/main-page/groups-tree-page/common/dependencies/visualization/graph-hops.ts

```diff
@@ -31,7 +31,7 @@ import { extractAllServices } from '../../../../utils/service-docs-tree-utils';
  *
  * In this scenario, we say that 2 hops are needed to reach service Y (PivotService --> TheEvent --> Y).
  * However, we say that the Pivot Service and X are not connected at all, leading to an infinite number of hops.
- * This is because we follow the graph in a directed way, so when looking an the consumed Events of a service, we then go to the services that produced these Events, and we do not look at services that consumed these Events.
+ * This is because we follow the graph in a directed way, so when looking an the subscribed events of a service, we then go to the services that produced these Events, and we do not look at services that consumed these Events.
```

Comment:
(I cannot comment directly on the affected line)

In the lines above, we talk about an example Event, and we still use the old wording. So my suggestion:

``` diff
* The hops are calculated by following the APIs/Events in a directed way.
* Example:
* Assume there is only one Event, and this Event is:
-* - Consumed by the Pivot Service and by service X
+* - Subscribed to by the Pivot Service and by service X
-* - Produced by service Y
+* - Published by service Y
```

Reply:
Addressed in [380ba05](https://github.com/osrgroup/msadoc/pull/75/commits/380ba05360e0fcb697ce53874c65ca76dfba6f0e)